### PR TITLE
Generate classes links

### PR DESCRIPTION
**Pull request by SemenchenkoVitaliy**
Closes: https://github.com/metarhia/metadoc/issues/66

### CODE REVIEW

File: lib/generator.js [hunk 1]
```diff
@@ -175,6 +175,14 @@ const genComment = (comment, opts, typeLine) => {
   return formatLine(line, 0, COMMENTS_SECONDARY_OFFSET);
 };
 
+const genClassLink = header =>
+  '#' +
+  header
+    .toLowerCase()
+    .replace(/^#*\s*/, '')
+    .replace(/[^\w\s-]/g, '')
```
Comment (lundibundi) [inline lib/generator.js]:
```suggestion
    .replace(/[^\w\s-]/, '')
```

File: lib/generator.js [hunk 2]
```diff
@@ -418,6 +431,12 @@ const generateMd = (inventory, options) => {
     if (!opts.removeInterface) {
       buf.push(headerify(`Interface: ${name}\n`, opts.interfaceLevel));
     }
+    opts.customClasses = new Map();
+    methods.forEach((sig, method) => {
+      if (sig.isClass) {
```
Comment (lundibundi) [inline lib/generator.js]: Nit: you can use .filter here.

Comment (SemenchenkoVitaliy) [reply lib/generator.js]: `methods` is a `Map`.

**Comment by lundibundi** [comment]
Landed in  d0263eba83115fa8882d6fb4d105c7c3c4cc2c5a.

**Pull request closed without merging**